include/kdbush.hpp

```diff
@@ -1,64 +1,89 @@
+// https://github.com/mourner/kdbush.hpp
+
 #pragma once
 
 #include <algorithm>
 #include <cmath>
 #include <vector>
 
+template<class TFloat = double, class TIndex = std::size_t>
```

Comment:
Should we call this `TFloat` rather than something like `TCoord`? It should work for integer types too in theory...